### DISCUSSION TITLE
Is it possible to run linux like commands on pockethost instance?

### DISCUSSION THREAD

**Discussion by rohitkrishna094** [pockethost/pockethost#388] [Q&A] [answered]
I deployed one instance and was able to copy my pb_data and pb_migrations folder into the instance through ftp using a tool like winscp.

But is there a way to "login" to the pockethost instance and then run terminal commands? I don't have any usecase per-se but was just curious if this was possible with regular FTP(s) or would it require pockethost to have ssh enabled(which I believe is not yet planned)?

**Comment by benallfree** [comment, marked as answer]
No, it's just FTP access. We are considering SSH access, but it introduces a lot of security issues. More info on Discord if you'd like to jump in there. https://discord.gg/nVTxCMEcGT
